assets/js/components/KeyMetrics/MetricTileHeader.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * MetricTileHeader component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Tooltip } from 'googlesitekit-components';
+import InfoIcon from '../../../svg/icons/info-green.svg';
+
+export default function MetricTileHeader( { title, infoTooltip } ) {
+	return (
+		<div className="googlesitekit-km-widget-tile__title-container">
+			<h3 className="googlesitekit-km-widget-tile__title">{ title }</h3>
```

Comment:
Any reason to use `title` over `children` here? `title` just comes with its own expectations as a prop name to me; it's used as an HTML attribute and we are using it for something different here: the contents (eg. `children`) of the header tag.

I think it might be better to use `children` here so the component is easier to understand in use from a glance.

----

I suppose the tiles themselves already sort of use `title` this way… I guess maybe it's better to be consistent within the hierarchy of these components, so it's best to leave it as-is (eg. keep the `title` attribute)… but I figured I'd leave the comment as food-for-thought/maybe a follow-up/cleanup issue idea later 😅